src/app/info/WikiItems.tsx

```diff
@@ -94,13 +115,98 @@ const WikiItems: React.FC<WikiItemsProps> = ({ rows }) => {
         const indexToRemove: number = displayRows.findIndex(
             (sortedRow) => row.wiki_key === sortedRow.wiki_key
         );
-        setDisplayRows((DisplayRows) => {
-            const temp = DisplayRows.slice();
+        setDisplayRows((displayRows) => {
+            const temp = displayRows.slice();
             temp.splice(indexToRemove, 1);
             return temp;
         });
         return indexToRemove;
     };
+
+    const [isSwapping, setIsSwapping] = useState(false);
```

Comment:
maybe rename this to isSwapMode to be similar to the isEditMode state used before. Completely up to you though, if this makes more sense go for it.